src/core/bptree_set_test.cc

```diff
@@ -302,11 +302,19 @@ TEST_F(BPTreeSetTest, MemoryUsage) {
   AllocType alloc(&mi_alloc);
   absl::btree_set<pair<double, sds>, std::greater<pair<double, sds>>, AllocType> btree(alloc);
 
-  LOG(INFO) << "btree before: " << mi_alloc.used() << " bytes";
+  ASSERT_EQ(0, mi_alloc.used());
   for (size_t i = 0; i < sds_vec.size(); ++i) {
     btree.emplace(i, sds_vec[i]);
   }
-  LOG(INFO) << "btree after: " << mi_alloc.used() << " bytes";
+  LOG(INFO) << "abseil btree: " << mi_alloc.used() << " bytes";
```

Comment:
```suggestion
  ASSERT_GT(mi_alloc.used(), 0) << mi_alloc.used();
```

Reply:
(to test that `btree_set` actually uses `alloc`)

Reply:
done